MonoZelda/Link/Projectiles/Arrow.cs

```diff
@@ -10,6 +10,7 @@ public class Arrow : Projectile, IProjectile
     private bool Finished;
     private float projectileSpeed = 4f;
     private int tilesTraveled;
+    private bool rotate;
```

Comment:
I'm a bit confused by what a bool value called rotate would mean. Rotate by 180 degrees? Maybe flip is a better name in that case? Or storing a direction itself when created could make more sense than reading the player's direction every update.